### PR TITLE
ALBS-297: Added compression build logs

### PR DESCRIPTION
**Pull request by anfimovdm**

### CODE REVIEW

**Review by soksanichenko — APPROVED** [review]

File: build_node/build_node_builder.py [hunk 1]
```diff
@@ -105,7 +106,7 @@ def run(self):
             ts = int(self.__start_ts.timestamp())
             task_dir = os.path.join(self.__working_dir, str(task.id))
             artifacts_dir = os.path.join(task_dir, 'artifacts')
-            task_log_file = os.path.join(task_dir, f'albs.{ts}.log')
+            task_log_file = os.path.join(task_dir, f'albs.{ts}.gz.log')
```
Comment (Korulag) [inline build_node/build_node_builder.py:109]: `.gz` extension should be at the end

File: build_node/build_node_builder.py [hunk 2]
```diff
@@ -165,6 +167,10 @@ def run(self):
                     build_artifacts = []
                     success = False
                 finally:
+                    with open(task_log_file, 'rb') as src:
```
Comment (soksanichenko) [inline build_node/build_node_builder.py:170]: more short:
``` with open(one_path) as fd_1, open(two_path) as fd_2:```

**Review by soksanichenko — APPROVED** [review]

File: build_node/build_node_builder.py (hunk 2 above)
Comment (anfimovdm) [reply build_node/build_node_builder.py:170]: it doesn't work in python 3.6.8

Comment (soksanichenko) [reply build_node/build_node_builder.py:170]: Are you sure?
```
[soksanichenko@localhost ~]$ python3.6
Python 3.6.8 (default, Nov  9 2022, 09:57:34)
[GCC 8.5.0 20210514 (Red Hat 8.5.0-15)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> with open('/tmp/1', 'w') as fd1, open('/tmp/2', 'w') as fd2:
...   fd1.write('1')
...   fd2.write('2')
...
1
1
>>>
[soksanichenko@localhost ~]$ cat /tmp/1
1[soksanichenko@localhost ~]$ cat /tmp/2
2[soksanichenko@localhost ~]$
```

**Review by maccelf — APPROVED** [review]

File: build_node/build_node_builder.py (hunk 2 above)
Comment (anfimovdm) [reply build_node/build_node_builder.py:170]: yup, you're right, I guess I did something wrong, thanks

**Review by Korulag — APPROVED** [review]

**Pull request merged**